### PR TITLE
Feat: Added the new dashboard page

### PR DESCRIPTION
**Pull request by Rohith1510**
### Added the new dashboard page

In this PR, I created new components for the dashboard page displayed after login.
Currently, the content is static, as usual.

Screenshot 2024-11-24 224149

### CODE REVIEW

File: app/Login/page.tsx [hunk 1]
```diff
@@ -9,14 +9,30 @@ export async function generateMetadata() {
...
 const LoginPage = () => {
-
   return (
-    <div className="flex justify-center items-center flex-wrap min-h-screen bg-white dark:bg-zinc-950 text-zinc-900 dark:text-white">
-      <div className="flex justify-center items-center">
-        <LoginAuth />
+    <div className="relative min-h-screen w-full">
+      {/* Background images with dark/light mode support */}
+      <div className="absolute inset-0 w-full h-full">
+        <img
```
Comment (Rithik2211) [inline app/Login/page.tsx]: Use Nextjs Image tag import from next js not like react  won't work here.

File: app/Login/page.tsx [hunk 2]
```diff
@@ -9,14 +9,30 @@ export async function generateMetadata() {
...
-        <LoginAuth />
+    <div className="relative min-h-screen w-full">
+      {/* Background images with dark/light mode support */}
+      <div className="absolute inset-0 w-full h-full">
+        <img 
+          src="/loginpage/login_bg_light.png" 
+          alt="Login background light" 
+          className="object-cover w-full h-full  block dark:hidden"
+        />
+        <img
```
Comment (Rithik2211) [inline app/Login/page.tsx]: Use Nextjs Image tag import from next js not like react  won't work here.

File: app/settings/page.tsx [hunk 3]
```diff
@@ -0,0 +1,225 @@
...
+        //   <Alert className="mb-6 bg-green-50 border-green-200">
+        //     <AlertDescription>Settings saved successfully!</AlertDescription>
+        //   </Alert>
+        )} */}
+
+        {/* Profile Section */}
+        <div className="mb-8">
+          <div className="flex items-center gap-6">
+            <div className="relative">
+              <img
```
Comment (Rithik2211) [inline app/settings/page.tsx]: Use Nextjs Image tag import from next js not like react  won't work here.

**Review by Rithik2211 — COMMENTED** [review]
change the requested Image chnages.

**Comment by Rohith1510** [comment]
Bug resolved @Rithik2211

**Review by Rithik2211 — APPROVED** [review]

**Pull request merged**